packages/@react-spectrum/table/src/TableView.tsx

```diff
@@ -937,7 +937,7 @@ function TableRowGroup({children, ...otherProps}) {
   let {rowGroupProps} = useTableRowGroup();
 
   return (
-    <div {...rowGroupProps} {...otherProps}>
+    <div {...rowGroupProps} {...otherProps} role="presentation">
```

Comment:
I don't think this is right. This component is also used for the table header row group. Plus, moving the row group to the scrollable element won't work when we have sections. We need a different solution.

Reply:
After chatting with @LFDanLu , this is a no go solution because Aria doesn't support nested row groups yet, so this would break our work on Table Sections. James is potentially submitting a bug/feature request against the spec, I'll link it here if he has. At this point in time, there appears to be nothing we can do about the TableView extra focus ring, and I believe the accessibility is worse for always announcing that there is only 1 row. So I'm going to undo the tabIndex change on the ScrollView and we'll have to deal with FF having an extra tab stop on our tables for now.